cli/run.go

```diff
@@ -17,6 +20,9 @@ import (
 var runDescription = "Create a container object in Pouchd, and start the container. " +
 	"This is useful when you just want to use one command to start a container. "
 
+// The default escape key sequence: ctrl-p, ctrl-q
+var defaultEscapeKeys = []byte{16, 17}
```

Comment:
I think this const can put into term package, since exec cmd or attach cmd could reuse it, wdyt? @xiaoxubeii

Reply:
@ZYecho Done.